docs/src/routes/docs/components/autocomplete.svx

```diff
@@ -93,6 +93,30 @@ Falsy values passed to classes will be disregarded.
 | **`change`** | `{ value }` | Fired when the selection changes. The `value` contains the selection array. |
 
 
+## Slots {#slots}
+
```

Comment:
How about you just mention that the same slots in `autocomplete-field` can be passed here, just to not duplicate any changes to the slots documentation?